### PR TITLE
Fixes the get name function.

### PR DESCRIPTION
**Pull request by mihai-vlc**
Fixes the get name function logic
as part of the course.

### CODE REVIEW

**Comment by mihai-vlc** [comment]
Post a comment.

File: src/main/java/ro/tefacprogramator/m4/listareproduse/ListareProduse.java [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 public class ListareProduse {
```
Comment (mihai-vlc) [inline src/main/java/ro/tefacprogramator/m4/listareproduse/ListareProduse.java:3]: poate redenumim clasa ?

**Comment by Dan-Clement** [comment]
5 stars

**Comment by FlorentinaBusca** [comment]
Nice work :)

**Comment by luchian27** [comment]
Votam pentru redenumirea clasei ?? :))

File: src/main/java/ro/tefacprogramator/m4/listareproduse/ListareProduse.java [hunk 2]
```diff
@@ -2,6 +2,7 @@
...
     public String getName() {
-        return "";
+        String nume = "Produs1";
```
Comment (luchian27) [inline src/main/java/ro/tefacprogramator/m4/listareproduse/ListareProduse.java]: Produs2

File: src/main/java/ro/tefacprogramator/m4/listareproduse/ListareProduse.java [hunk 3]
```diff
@@ -2,6 +2,7 @@
...
+        return nume;
```
Comment (FlorentinaBusca) [inline src/main/java/ro/tefacprogramator/m4/listareproduse/ListareProduse.java:6]: :)

**Pull request merged**